extensions/amp-consent/0.1/tcf-api-commands.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {TCF_POST_MESSAGE_API_COMMANDS} from './consent-info';
+import {isEnumValue, isObject} from '../../../src/types';
+import {user} from '../../../src/log';
+
+const TAG = 'amp-consent';
+
+/**
+ * @param {!Object} payload
+ */
+export function handleTcfCommand(payload) {
```

Comment:
nit: change the param to command (See Principle of Least Knowledge)

Reply:
We will need the full payload in the next PR.